libs/checksum/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,7 +15,7 @@
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
 
 <!-- create the bean -->
-<bean id="checksumProvider" class="org.codice.ddf.checksum.impl.Adler32ChecksumProvider">
+<bean id="checksumProvider" class="org.codice.ddf.checksum.impl.Sha256ChecksumProvider">
```

Comment:
( ❓ ) The Adler32 version is faster. Do we want to replace all instances of it with the Sha256 version or let callers decide which one they want?

Reply:
I'm in favor of making it configurable with Adler32 being the default.